### ISSUE TITLE
Initial users appears to be broken on the UserMultiSelectMenu

### ISSUE THREAD

**Issue by kigsmtua** [nicklambourne/slackblocks#71]
```python
    multi_select_user = UserMultiSelectMenu(
        action_id="multi_users_select",
        placeholder=Text("Select one or more users", type_=TextType.PLAINTEXT),
        initial_users=["U064B5H1309", "U063JR973UP"],
    )
   ```
   
   Given the above piece of code,  looks like the initial_users fails to render with the below error
   
```
    user_multi_select["initial_users"] = [
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

.0 = <list_iterator object at 0x7fdaa37a6b00>

    user_multi_select["initial_users"] = [
>       initial_option._resolve() for initial_option in self.initial_users
    ]
E   AttributeError: 'str' object has no attribute '_resolve'

```

This looks to be a case of the initial_users being specified as a list of strings, but we try to call the _resolve method from all the elements in intial_users, which will of course fail.

I am happy to push a fix for this.

**Comment by nicklambourne** [comment]
Hey @kigsmtua, looks to me like you're right and this wasn't picked up by the tests. Happy to do a new release if you want to send through a PR.

**Comment by nicklambourne** [comment]
These changes have been included in `v1.0.4`, thanks for your help fixing this issue!

https://github.com/nicklambourne/slackblocks/releases/tag/v1.0.4
